### PR TITLE
Get audit logs

### PR DESCRIPTION
**Pull request by YoavBZ**
Add get_audit_logs method
https://docs.fireblocks.com/api/swagger-ui/#/Audit%20Logs/get_audits

### CODE REVIEW

File: fireblocks_sdk/api_types.py [hunk 1]
```diff
@@ -42,18 +43,28 @@ def __init__(self, peer_type, peer_id=None, one_time_address=None):
 ONE_TIME_ADDRESS = "ONE_TIME_ADDRESS"
 TYPED_MESSAGE = "TYPED_MESSAGE"
 
-TRANSACTION_TYPES = (TRANSACTION_TRANSFER, TRANSACTION_MINT, TRANSACTION_BURN, TRANSACTION_SUPPLY_TO_COMPOUND, TRANSACTION_REDEEM_FROM_COMPOUND, RAW, CONTRACT_CALL, ONE_TIME_ADDRESS, TYPED_MESSAGE)
+TRANSACTION_TYPES = (
```
Comment (zoharsf) [inline fireblocks_sdk/api_types.py:46]: Good

File: fireblocks_sdk/sdk.py [hunk 2]
```diff
@@ -1686,6 +1683,16 @@ def get_off_exchanges(self):
...
 
+    def get_audit_logs(self, time_period: TimePeriod):
+        """
+        Get audit logs
+        :param time_period: The last time period to fetch audit logs
+        """
+
+        url = f"/v1/audits"
+
+        return self._get_request(url, query_params={'timePeriod': time_period})
```
Comment (zoharsf) [inline fireblocks_sdk/sdk.py:1694]: Is the timePeriod query param really in camelCase? 🤦🏻‍♂️

**Review by zoharsf — APPROVED** [review]
LGTM

File: fireblocks_sdk/sdk.py (hunk 2 above)
Comment (YoavBZ) [reply fireblocks_sdk/sdk.py:1694]: That's right

File: fireblocks_sdk/api_types.py [hunk 3]
```diff
@@ -262,11 +262,38 @@ def __init__(self, total_amount_larger_than=None, asset_id=None, order_by=None,
         self.after = after
 
 
-class NFTOwnershipStatusValues(Enum):
+class GetOwnedNftsSortValues(str, Enum):
```
Comment (zoharsf) [inline fireblocks_sdk/api_types.py:265]: Please make sure this doesn't break backwards compatibility

**Review by zoharsf — APPROVED** [review]

File: fireblocks_sdk/api_types.py (hunk 3 above)
Comment (YoavBZ) [reply fireblocks_sdk/api_types.py:265]: It doesn't (actually the usage was wrong until now)

**Pull request merged**